server/dataset/cxg_dataset.py

```diff
@@ -21,9 +21,13 @@
 
 
 class CxgDataset(Dataset):
-    # TODO:  The tiledb context parameters should be a configuration option
+    # These defaults are overridden by the config variable: server.adaptor.cxg_adaptor.tiledb_cxt
     tiledb_ctx = tiledb.Ctx(
-        {"sm.tile_cache_size": 8 * 1024 * 1024 * 1024, "sm.num_reader_threads": 32, "vfs.s3.region": "us-east-1"}
+        {
+            "sm.tile_cache_size": 8 * 1024 ** 3,
+            "py.init_buffer_bytes": 16 * 1024 ** 3,
+            "vfs.s3.region": "us-east-1",
```

Comment:
I was wondering why the region is even required. Our buckets live in `us-west-2` so it looks like this is ignored by TileDB.

Reply:
The idea was just to have an explicit default in the code so we didn't have to rely on a TileDB default.  You are right that the current configurations override it to match our current deployment.  As or deployment evolves (eg, multi-region?), the config could change.